src/app/store/balances/epics.ts

```diff
@@ -0,0 +1,79 @@
+import { TezosToolkit } from '@taquito/taquito';
+import { BigNumber } from 'bignumber.js';
+import { combineEpics, Epic } from 'redux-observable';
+import { catchError, forkJoin, from, map, Observable, of, switchMap } from 'rxjs';
+import { Action } from 'ts-action';
+import { ofType, toPayload } from 'ts-action-operators';
+
+import { fecthTezosBalanceFromTzkt, fecthTokensBalancesFromTzkt } from 'lib/apis/tzkt/api';
+import { fetchBalance, fetchTezosBalanceAtomic, toTokenSlug } from 'lib/temple/assets';
+import { atomsToTokens } from 'lib/temple/helpers';
+import { IAccountToken } from 'lib/temple/repo';
+
+import { loadTokensBalancesFromChainAction, loadTokensBalancesFromTzktAction } from './actions';
+
+const YUPANA_TOKENS = [
+  'KT1Rk86CX85DjBKmuyBhrCyNsHyudHVtASec_4',
+  'KT1Rk86CX85DjBKmuyBhrCyNsHyudHVtASec_0',
+  'KT1Rk86CX85DjBKmuyBhrCyNsHyudHVtASec_5'
+];
+const YUPANA_MULTIPLIER = 18;
```

Comment:
If `loadTokensBalancesFromTzktEpic` works properly with Yupana tokens, consider renaming it to `YUPANA_TOKENS_DECIMALS`. Otherwise, consider using decimals from these fragments of metadata: https://github.com/madfish-solutions/templewallet-extension/pull/847/files#diff-01a90946e02c3c9ab437b5b0015b3512c721d2921c3ed599a88b6152e19b217fR174

Reply:
It is not Yupana decimals, it is Yupana multiplier. It is different things